lxd/main.go

```diff
@@ -947,6 +953,18 @@ they otherwise would.
 		}
 	}
 
+	if imagesAutoUpdate {
+		_, err = c.SetServerConfig("images.auto_update_cached", "true")
```

Comment:
Oh right, that would work, except that as I commented, we shouldn't be setting images.auto_update_cached in the first place and the key we should set is an integer :)